docs/anatomy.md

```diff
@@ -0,0 +1,23 @@
+# Anatomy of a deep learning library
+
+The fastai library combines the best elements of software automation and modern deep learning. The deep learning is principles are well covered by the course material, but the design of the code itself also reveals a lot of insights into the practical matters of putting it all together.
+
+## Working with data
+
+A substantial amount of the code in the fastai library is dedicated to reading, writing, reshaping, polishing, and otherwise working with data. There are a few places where this sort of logic is concentrated.
+
+* [core.py](../fastai/core.py) contains a number of helpers for translating your data into common shapes like `np.array`, `torch.Tensor`, or `torch.autograd.Variable`.
+* [dataloader.py](../fastai/dataloader.py) is a simplified variant of `torch.util.data.DataLoader`. This `DataLoader` iterates over a `Dataset` in batches rather than one example at a time. There are also conveniences here for working with a few differently shaped inputs that are common in practice.
```

Comment:
Key difference isn't that it's simplified, but that it's faster, since it uses threads, not processes.